README.md

```diff
@@ -0,0 +1,52 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+<p align="center">Desktop view</p>
+<img src="./images/web.jpg">
+
+<p align="center"> Mobile view</p>
+<img src="./images/mobile.jpg"> 
+
+<h1 align="center"> Module One Capstone Project </h1>
+
+> This is a repository for my Module One Capstone Project at Microverse. It is a website for a fictitious, upcoming F1 Sustainability Summit.
+> Visit the [Live Link](https://benmuiruri.github.io/f1-sustainability-capstone/) to view the website.
+## Built With
+
+- **HTML** and **CSS** and **Javascript**
+
+## Getting Started
+
+To get a local copy up and running follow these setup steps below.
+
+### Setup
+
+- Simply fork this project and it should be ready for you to edit as you see fit.
+- In case you run into any errors please raise an [issue](https://github.com/Benmuiruri/f1-sustainability-capstone/issues).
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/Benmuiruri)
+- Twitter: [@twitterhandle](https://twitter.com/_optimize)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/benjamin-kiarie-180b66149/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Benmuiruri/f1-sustainability-capstone/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+## Video Presentation
+
```

Comment:
- Well done on the presentation, I really like it. You were too specific and too clear. You manage to talk about everything that you used and we saw responsiveness on both screens. What I would suggest you in the next presentation is to show us a piece of code that you find interesting or you are proud of. Otherwise, it was amazing 👏 💪

Reply:
Thanks, I'll write a script before taking the video to practice what I will say within the 5 minutes and ensure I include a bit of code. Thanks for the feedback